vaporetto/src/kytea_model.rs

```diff
@@ -459,12 +460,24 @@ impl TryFrom<KyteaModel> for Model {
         let mut type_ngrams = vec![];
         for (type_ngram, v) in type_dict.dump_items() {
             let weight_size = config.type_w as usize * 2 - type_ngram.len() + 1;
+            let mut ngram = type_ngram
+                .into_iter()
+                .collect::<String>()
+                .as_bytes()
+                .to_vec();
+            for t in &mut ngram {
+                *t = match *t {
+                    b'D' => CharacterType::Digit as u8,
+                    b'R' => CharacterType::Roman as u8,
+                    b'H' => CharacterType::Hiragana as u8,
+                    b'T' => CharacterType::Katakana as u8,
+                    b'K' => CharacterType::Kanji as u8,
+                    b'O' => CharacterType::Other as u8,
+                    t => t,
```

Comment:
What does Vaporetto expect in this case?
I think an error should be returned when character types other than Vaporetto expects are given.

Reply:
fixed